pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -75,6 +76,9 @@ func NewFramework(r Registry, _ *runtime.Unknown) (Framework, error) {
 		if pp, ok := p.(PrebindPlugin); ok {
 			f.prebindPlugins = append(f.prebindPlugins, pp)
 		}
+		if pp, ok := p.(PostbindPlugin); ok {
```

Comment:
Please rebase and follow the same model for initializing plugins.

Reply:
DONE